bitmap-team-deuces/lib/notes.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
This file seems useful during development, but since this is a PR it is unnecessary since it doesn't relate to the program, please remove it from the PR :)